test/e2e/storage/csi_objects.go

```diff
@@ -70,14 +69,30 @@ func csiDriverRegistrarClusterRole(
 	}
 	superuserClientset, err := clientset.NewForConfig(rc)
 	framework.ExpectNoError(err, "Failed to create superuser clientset: %v", err)
-	By("Creating the CSI driver registrar cluster role")
+	By(fmt.Sprintf("Creating the %s cluster role", role.ObjectMeta.Name))
```

Comment:
nit: role.GetName() instead of role.ObjectMeta.Name

Reply:
Fixed as suggested.